### PR TITLE
fix(displays): excessive screen brightness when pixels not visible

### PR DESCRIPTION
**Pull request by tyler58546**
Fixes #[issue_no]

## Summary of Changes

Pixels make the screen appear slightly darker. To compensate for this, the brightness of the screens was increased. In #5483, the pixel effect is removed when not zoomed in on the screen, however the brightness is still increased. This PR fixes this issue by adding a translucent black overlay on the screens when the pixels are not visible.

## Additional context

Discord username (if different from GitHub):

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Comment by bouveng** [comment]
I think it is better and so much easier to just lower the emissive for screens to a sane level.

An over powered emissive starts to bleed and text get artifacts. It does not solve the issue with intensity at 0.1 0.2 0.3 0.4 0.5 lower level steps, where control is more important.

The highest 100% (8.0) setting is useless.

This is how the max setting and its bleed originally was implemented and looked.

image

**Comment by davidwalschots** [comment]
I'd suggest closing this PR and having @bouveng provide an improvement PR.

File: flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/A320_Neo_EICAS.js [hunk 1]
```diff
@@ -152,6 +153,7 @@ class A320_Neo_EICAS extends Airliners.BaseEICAS {
             return;
         }
         this.lcdOverlay.style.opacity = SimVar.GetSimVarValue("L:A32NX_MFD_MASK_OPACITY", "number");
+        this.brightnessCompensation.style.opacity = 1 - SimVar.GetSimVarValue("L:A32NX_MFD_MASK_OPACITY", "number");
```
[the remark this answers is not in the document]
Comment (ErickSharp) [reply flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/A320_Neo_EICAS.js:156]: Isn't it necessary to read the simvar several times in order to calculate the opacity?

**Comment by 2hwk** [comment]
Closing as covered by #5661

**Pull request closed without merging**